src/main/sensors/voltage.c

```diff
@@ -182,8 +199,25 @@ void voltageMeterADCRead(voltageSensorADC_e adcChannel, voltageMeter_t *voltageM
 {
     voltageMeterADCState_t *state = &voltageMeterADCStates[adcChannel];
 
-    voltageMeter->filtered = state->voltageFiltered;
+    voltageMeter->displayFiltered = state->voltageDisplayFiltered;
     voltageMeter->unfiltered = state->voltageUnfiltered;
+#if defined(USE_BATTERY_VOLTAGE_SAG_COMPENSATION)
+    voltageMeter->sagFiltered = state->voltageSagFiltered;
+#endif
+}
+
+bool isSagCompensationConfigured(void)
+{
+    bool isConfigured = false;
+#if defined(USE_BATTERY_VOLTAGE_SAG_COMPENSATION)
+    for (unsigned i = 0; i < PID_PROFILE_COUNT; i++) {
+        if (pidProfiles(i)->vbat_sag_compensation > 0) {
+            isConfigured = true;
+        }
+    }
```

Comment:
We could save a little processing time if we used a static initialized in `voltageMeterADCInit` and just made `isSagCompensationConfigured()` a getter function. Or even eliminate the getter function and just reference the static if only used here.